### DISCUSSION TITLE
Feature Request: Cody Adapter

### DISCUSSION THREAD

**Discussion by KarlEricsson** [olimorris/codecompanion.nvim#345] [Ideas]
Thanks for a great plugin! I would love a Cody adapter added if possible. Especially now since [sg.nvim](https://github.com/sourcegraph/sg.nvim) is abandoned. It think it should be technically possible with [this](https://www.npmjs.com/package/@sourcegraph/cody). Hopefully it's not too much work :)

**Comment by olimorris** [comment]
Would welcome a PR and corresponding tests for it. I've written a whole guide on how to do it [here](https://github.com/olimorris/codecompanion.nvim/blob/main/doc/ADAPTERS.md) and someone from the community added an [adapter](https://github.com/olimorris/codecompanion.nvim/pull/329) last week
